client/reducers/reducer-businesses.js

```diff
@@ -23,12 +27,47 @@ export default function businesses(state = [], action = {}) {
       });
 
     case 'DELETE_BUSINESS':
-      return state.filter(item => item.id !== action.businessId);
+      return state.filter(business => business.id !== action.businessId);
 
     case 'SET_BUSINESSES':
-      return action.businesses;
+      return {
+        businesses: action.businesses.rows,
+        count: action.businesses.count
+      };
 
     default:
       return state;
   }
 }
+
+
+// const initialState = {
+//   businesses: [],
+//   count: 0
+// }
+// export default function businesses(state =initialState, action = {}) {
+//   switch (action.type) {
+//     case 'ADD_BUSINESS':
+//       return [
+//         ...state,
+//         action.business
+//       ];
+
+//     case 'UPDATE_BUSINESS':
+//       return state.map((business) => {
+//         if (business.id === action.updatedBusiness.id) {
+//           return action.business;
+//         }
+//         return business;
+//       });
+
+//     case 'DELETE_BUSINESS':
+//       return state.filter(business => business.id !== action.businessId);
+
+//     case 'SET_BUSINESSES':
+//       return action.businesses;
+
+//     default:
+//       return state;
+//   }
+// }
```

Comment:
Good work @jherey,  but you need to make little changes in your business reducer file, I noticed you are mutating your state, you can change this by returning your previous state. Also, remove commented code so that your codebase will look nicer and more professional.

Reply:
Ok, I'll remove commented codes and refactor my code to stop mutating my state.